INSTALLING.md

```diff
@@ -2,199 +2,266 @@
 
 These instructions will guide you through the installation of Tenstorrent system tools and drivers, followed by the installation of TT-Metalium and TT-NN.
 
-> [!IMPORTANT]
->
-> If you are using a release version of this software, please check the installation instructions packaged with that version. You can find them in either the release assets for that version, or in the source files for that version tag.
-
 ---
 
-## Installation Steps
+## Prerequisites:
 
-### Step 1. Driver & Firmware
+### 1: [Setup the Hardware](https://docs.tenstorrent.com/quickstart.html#unboxing-and-hardware-setup)
 
-Follow the Software Setup instructions for your specific board or system provided on our [general docs](https://docs.tenstorrent.com).
+Once you have setup the hardware move on to step 2.
 
-If you have purchased a Grayskull card, you will find the instructions [here](https://docs.tenstorrent.com/aibs/grayskull/installation.html).
+---
 
-Note the current compatability matrix:
+### 2: Install Driver & Firmware
 
-| Device              | OS              | Python   | Driver (TT-KMD)    | Firmware (TT-Flash)                        | TT-SMI                | TT-Topology                    |
-|---------------------|-----------------|----------|--------------------|--------------------------------------------|-----------------------|--------------------------------|
-| Grayskull           | Ubuntu 20.04    | 3.8.10   | v1.27.1            | fw_pack-80.9.0.0 (v80.9.0.0)               | v2.2.0 or above       | N/A                            |
-| Wormhole            | Ubuntu 20.04    | 3.8.10   | v1.27.1            | fw_pack-80.10.0.0 (v80.10.0.0)             | v2.2.0 or above       | N/A                            |
-| T3000 (Wormhole)    | Ubuntu 20.04    | 3.8.10   | v1.27.1            | fw_pack-80.10.0.0 (v80.10.0.0)             | v2.2.0 or above       | v1.1.3 or above, `mesh` config |
+#### Install System-level Dependencies
+```
+wget https://raw.githubusercontent.com/tenstorrent/tt-metal/refs/heads/main/install_dependencies.sh
+chmod a+x install_dependencies.sh
+sudo ./install_dependencies.sh
+```
 
 ---
 
-### Step 2. System-level dependencies
+#### Install the Driver (TT-KMD)
```

Comment:
You copy pasted these from the other pages?

I think there needs to be more synchronization between our docs' team and the customer docs' team. There seems to be separate tracks of work trying to do the same thing. Let's set up some time between @zzigler-tt  and @bbeggsTT to work out a game plan and be clear with what our goals are.

I know the deliverable we want to provide a seamless experience for customers who are less hacker-inclined. I'll caution everyone in saying a couple points:

1) Multiple people + teams in TT want the same thing
2) For this particular deliverable, trying to most fast but separately from the organization will only result in a documentation that will leave customers still confused.
3) As a side note, copy-pasting documentation like this is something that SysEng (@warthog9 ) has specifically requested we *stop* and have an organization plan for.

Reply:
We included that part here because it is relatively short and considering @adifrancescoTT  points about creating self contained installation guides and cutting down on amount of clicks/link hopping needed through an installation procedure.

From @adifrancescoTT's presentation here: https://docs.google.com/presentation/d/1xT3UonEg3u_UHP2ehaBAZwRhKt30b2RlenmHyG10PRk/edit#slide=id.g31b7042d5d0_0_73

We did consult @warthog9 about including the TT-KMD section, I can add you to that conversation in slack.

Our options when including relevant information from other locations are:
- Link the Content - Creates disjointed feeling in documentation by going between too many locations for a single install procedure. Cuts down on document length.
- Include the Content - Cuts down on the amount of link/location hopping but will force us to keep track of updates in other locations as well our own. Can also make documents much longer and daunting.
- Include Small Sections with Link - This is the approach we took for this section. The relevant info is included but with a reference to the full content.
- Alternative Option - Can we create a custom action in GitHub that allows us to "snapshot" or "include" content from other locations? Something that auto-updates if the original content changes?

Hopefully this provides some clarity on our intentions for these changes. All feedback is welcome.

Reply:
Sounds good, if you consulted with both @adifrancescoTT and @warthog9 's teams about this, then I'm good with approach 3.